docs/development.md

```diff
@@ -38,10 +15,8 @@ $ make binary
 The commands below require that you have $GOPATH. We highly recommended you put Kompose' code into your $GOPATH.
 
 ```console
-mkdir -p $GOPATH/src/github.com/kubernetes-incubator
-cd $GOPATH/src/github.com/kubernetes-incubator
-git clone https://github.com/$YOUR_GITHUB_USERNAME/kompose.git
-cd kompose
+git clone https://github.com/$YOUR_GITHUB_USERNAME/kompose.git $GOPATH/src/github.com/kubernetes-incubator/kompose
```

Comment:
>this will fail as there is no folder created.

`man git clone` gives following
```
git clone <repository> [<directory>]

<directory> = The name of a new directory to clone into. The "humanish" part of the source repository is used if no directory is explicitly given (repo for /path/to/repo.git and foo for host.xz:foo/.git). Cloning into an existing directory is only allowed if the directory is empty.
```

>although `kompose` is no `go get`able, it may be good to simply put `go get github.com/kubernetes-incubator/kompose` here with the relevant docs.

kompose is `go get`able